src/oss/confluent.sh

```diff
@@ -927,10 +927,13 @@ validate_java_version() {
     # 9.0.4 -> java version "9.0.4"
     # 10 -> java version "10" 2018-03-20
     # 10.0.1 -> java version "10.0.1" 2018-04-17
-    # We need to match to the end of the line to prevent sed from printing the characters that do not match
+    # Need to match the end of the line to prevent sed from printing the characters that do not match
     local java_version=$(java -version 2>&1 | sed -E -n 's/.* version "([0-9]*).*$/\1/p')
-    # Warn users if it is not Java7 or Java8, or if we can't figure it out.
-    if [[ "${java_version}" -ge "9" ]]; then
+    if [[ "${java_version}" -eq "1" ]]; then
+        java_version=$(java -version 2>&1 | sed -E -n 's/.* version "1.([0-9]*).*$/\1/p')
+    fi
+
+    if [[ "${java_version}" -ge "9" || "${java_version}" -lt "8" ]]; then
```

Comment:
This is not sufficient because this method is not called for `kafka`. 
But kafka is dropping support for java 7 as well (and we should add zk in the check for java 7 in 5.0.0 and beyond, doesn't make sense to support ZK only).